.github/workflows/checks.yml

```diff
@@ -14,19 +14,5 @@ jobs:
       with:
         python-version: '3.8.10'
     - uses: pre-commit/action@v2.0.3
-
     - uses: actions/checkout@v2
     - uses: luisremis/find-trailing-whitespace@master
-
```

Comment:
why is this check going away? does PEP8 check already checks for trailing spaces?

Reply:
I thought that pre-commit is something generic to run multiple github workflows. If you look at this PR, the trailing spaces seems to have run. Actually let's keep things simple for now, and let me reintroduce the previous configuration till I know of any benefits by doing it this way. Right now can't think of any apart from lesser code.

Reply:
oh, never mind. I see what you are saying. this change is ok, we were actually running it twice before

Reply:
phew! that's what I thought was happening - but I've not used the actions a lot.